README.md

```diff
@@ -747,8 +747,47 @@ peered with.
 
 `peer_region` defines the region of the remote VPC if it is not in the same region as Aiven VPC.
 
-Computed property `state` tells the current state of the VPC. This property cannot be
-set, only read.
+`state` is the state of the peering connection. This property is computed by Aiven 
+therefore cannot be set, only read. Where state can be one of: `APPROVED`, 
+`PENDING_PEER`, `ACTIVE`, `DELETED`, `DELETED_BY_PEER`, `REJECTED_BY_PEER` and 
+`INVALID_SPECIFICATION`. 
+
+Happy path of the `state` includes the following transitions of a VPC peering connection: 
+`APPROVED` -> `PENDING_PEER` -> `ACTIVE`.
+
+More details regarding each state:
+
+- `APPROVED` is the initial state after the user does a successful creation of a 
+peering connection resource via Terraform. 
+
+- `PENDING_PEER` appears when Aiven system has created one or more cloud resources from 
+the Aiven cloud account to the user's VPC/network that was specified by the peering 
+connection. This state means a VPC peering connection resource was successfully created 
+and in the pending state, and the user needs to perform a cloud provider-specific steps 
+in their cloud account to activate it. The state can change to `ACTIVE`, `DELETED`, 
+`DELETED_BY_PEER` and `REJECTED_BY_PEER`.
+
+- `ACTIVE` stands for successful VPC peering connection that has been established, and all 
+related cloud resource is in a state that allows traffic to flow to the peered network, 
+a successful VPC peering connection. VPC peering connection stays in this state in until it 
+gets deleted. State transitions are to `DELETED` and `DELETED_BY_PEER`.
+
+- `DELETED` means a user deleted the peering connection through the Aiven Terraform provider, 
+or Aiven Web Console or directly via Aiven API.
+
+- `DELETED_BY_PEER` appears when a user deleted the VPC peering connection through their cloud 
+account. That is, the user deleted the peering cloud resource in their account. There are no 
+transitions from this state
+
+- `REJECTED_BY_PEER` an AWS specific state, when VPC peering connection was in the `PENDING_PEER` state, 
+and the user rejected the AWS peering connection request.
+
+- `INVALID_SPECIFICATION` a VPC peering connection was in state APPROVED state. Still, something 
+the user controls broke a successful connection, for example, the peer cloud account of VPC 
```

Comment:
Perhaps something more along the lines of "`INVALID_SPECIFICATION` is a VPC peering connection that was in the `APPROVED` state but could not be successfully created because of something in the user's control" and then the examples